aten/src/ATen/core/TensorMethods.h

```diff
@@ -1159,6 +1156,9 @@ inline Tensor Tensor::cholesky(bool upper) const {
 inline Tensor Tensor::cholesky_solve(const Tensor & input2, bool upper) const {
     return type().cholesky_solve(*this, input2, upper);
 }
+inline std::tuple<Tensor,Tensor> Tensor::solve(const Tensor & A) const {
```

Comment:
Is this file autogenerated?

Reply:
This file is auto-generated separately, and then compared against this file. If this is not modified, then an error is thrown during the build process.